pkg/kudoctl/resources/dependencies/resolve_test.go

```diff
@@ -16,25 +16,25 @@ import (
 )
 
 type nameResolver struct {
-	Pkgs []packages.Package
+	Prs []packages.Resources
 }
 
 func (resolver nameResolver) Resolve(
 	name string,
 	appVersion string,
-	operatorVersion string) (*packages.Package, error) {
-	for _, pkg := range resolver.Pkgs {
-		if pkg.OperatorName() == name &&
-			(operatorVersion == "" || pkg.OperatorVersionString() == operatorVersion) &&
-			(appVersion == "" || pkg.AppVersionString() == appVersion) {
-			return &pkg, nil
+	operatorVersion string) (*packages.Resources, error) {
+	for _, pr := range resolver.Prs {
+		if pr.Operator.Name == name &&
+			(operatorVersion == "" || pr.OperatorVersion.Spec.Version == operatorVersion) &&
+			(appVersion == "" || pr.OperatorVersion.Spec.AppVersion == appVersion) {
```

Comment:
Maybe move the `OperatorVersionString` and `AppVersionString` funcs to the `packages.Resources` type? I think they're only used here